exist-core/src/test/java/org/exist/xmldb/SerializationTest.java

```diff
@@ -114,6 +114,12 @@ private final String getBaseUri() {
 			"<!DOCTYPE bookmap PUBLIC \"-//OASIS//DTD DITA BookMap//EN\" \"bookmap.dtd\">\n" +
 			"<bookmap id=\"bookmap-1\"/>";
 
+	private static final XmldbURI TEST_XML_DOC_WITH_XMLDECL_URI = XmldbURI.create("test-with-xmldecl.xml");
```

Comment:
Can this static field moved as suggested by Codacy?

Reply:
As requested I just appended a commit to address this.